package.json

```diff
@@ -8,10 +8,16 @@
     "node": "12"
   },
   "scripts": {
-    "start": "NODE_ENV=development webpack-dev-server --mode development --hot --host 0.0.0.0 --disable-host-check",
+    "start": "npm run start:swap-v1",
```

Comment:
shouldn't these be using `yarn` :thinking:

Reply:
all the others use npm. This way work with both